loader_hub/mondaydotcom/base.py

```diff
@@ -0,0 +1,89 @@
+"""monday.com reader."""
+from typing import List
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+import json
+import requests
+
+class MondayReader(BaseReader):
+    """monday.com reader. Reads data by a GraphQL query.
+
+    Args:
+        api_key (str): monday.com API key.
+    """
+    
+    def __init__(self, api_key: str) -> None:
+        """Initialize monday.com reader."""
+        
+        self.api_key = api_key
+        self.api_url = "https://api.monday.com/v2"
+
+    def parse_item_values(self, cv):
```

Comment:
more of a nit since we don't epxlicitly enforce this yet, but would be nice to explicitly type return values of these functions